etc/schema.json

```diff
@@ -6017,36 +6018,51 @@
                     "type": "object",
                     "properties": {
                         "active": {
+                            "description": "Number of currently active flows",
                             "type": "integer"
                         },
                         "emerg_mode_entered": {
+                            "description": "Number of times emergency mode was entered",
                             "type": "integer"
                         },
                         "emerg_mode_over": {
+                            "description": "Number of times recovery was made from emergency mode",
                             "type": "integer"
                         },
                         "get_used": {
+                            "description": "Number of reused flows",
                             "type": "integer"
                         },
                         "get_used_eval": {
+                            "description":
+                                    "Number of attempts at getting a flow directly from the hash",
                             "type": "integer"
                         },
                         "get_used_eval_busy": {
+                            "description":
+                                    "Number of times a flow was found in the hash but the lock for hash bucket could not be obtained",
                             "type": "integer"
                         },
                         "get_used_eval_reject": {
+                            "description":
+                                    "Number of flows that were successfully retrieved from hash but were still alive",
                             "type": "integer"
                         },
                         "get_used_failed": {
+                            "description":
+                                    "Number of times retrieval of flow from hash was attempted but was unsuccessful",
                             "type": "integer"
                         },
                         "icmpv4": {
+                            "description": "Number of ICMPv4 flows",
                             "type": "integer"
                         },
                         "icmpv6": {
+                            "description": "Number of ICMPv6 flows",
                             "type": "integer"
                         },
                         "memcap": {
+                            "description": "Number of times memcap was reached for flows",
```

Comment:
Really, not the memcap itself ?

Reply:
I think so. See: https://github.com/OISF/suricata/blob/master/src/flow-hash.c#L712

Besides, the `memcap` itself is a setting in `suricata.yaml` so always accessible there.
And, there's the `memuse` stat that tells how much memory is currently in use.
Lmk wdyt?